### ISSUE TITLE
Add Example Usage

### ISSUE THREAD

**Issue by vadim-hleif** [cloudposse/terraform-aws-ec2-bastion-server#11]
## what
* Add example invocation

## why
* We need this so we can soon enable automated continuous integration testing of module

**Comment by dalekurt** [comment]
```
module "bastion" {
  source                      = "./modules/aws-ec2-bastion-server"
  namespace                   = "${var.namespace}"
  stage                       = "${var.stage}"
  name                        = "${var.name}"
  tags                        = "${var.tags}"
  allowed_cidr_blocks_bastion = ["${var.allowed_cidr_blocks_bastion}"]
  vpc_id                      = "${module.vpc.vpc_id}"
  subnets = ["${lookup(module.public_subnets.az_subnet_ids,"us-east-1a")}"]
  security_groups = ["${module.vpc.vpc_default_security_group_id}"]
  ssh_user        = "${var.ssh_user}"
  key_name        = "${var.key_name}"
}
```

**Comment by osterman** [comment]
Closed by #27 which adds 0.12 support with tests
